### DISCUSSION TITLE
Extend mbsubmit to run after initial import

### DISCUSSION THREAD

**Discussion by arsaboo** [beetbox/beets#4455] [Ideas]
Right now the `mbsubmit` plugin only works at the time of import. I am not sure if this is by design or just an oversight. It makes sense to be able to create a tracklist to be submitted even after the initial import. 

Ideally, `mbsubmit` should be able to accept a query and print the tracklist to be submitted to MB.

**Comment by sampsyo** [comment]
Seems like an extremely reasonable extension (and not too challenging!) to me.

**Reply by arsaboo** [reply]
Indeed. I added another PR #4456 to your queue :)
